rsvp/setup.go

```diff
@@ -355,6 +367,29 @@ func (s *SetupSession) sendMessage(msgf string, args ...interface{}) {
 	}
 }
 
+func (s *SetupSession) sendInitialMessage(data *dcmd.Data, msgf string, args ...interface{}) {
+	switch data.TriggerType {
+	case dcmd.TriggerTypeSlashCommands:
+		content := "[RSVP Event Setup]: " + fmt.Sprintf(msgf, args...)

```

Comment:
you can use `data.SendFollowupMessage(reply interface{}, allowedMentions discordgo.AllowedMentions)` to handle things a bit easier